### ISSUE TITLE
[EN][dungeon-cata] Dungeon deserter

### ISSUE THREAD

**Issue by Frainzil** [Atlantiss/BugTracker#2419]
So i was doing Deadmines with a guild group 5/5. When we reached the 2nd boss; during the encounter every member within the party got a crash. Now after relogging i got the bug that i couldn't see the party, but still be in it. So trying to fix that i left the dungeon, but ofc i received the dungeon deserter. Now as far as i know i don't think a dungeon deserter should be still given when you are like 15min in a dungeon already? I think on retail there was a similar way about a timer being up, if it runs out people who leave wouldn't get the deserter since they were in the dungeon already for quiet some time. Image that there is a dungeon with the leader of the group being the healer and he dc. The other 4 are practically screwed since they cannot leave neither kick. What are they supposed to do?

**Comment by Jenovas** [comment]
Yes you are right - being in the dungeon for "More than 11 minutes = NO debuff".

**Comment by Arcadia713** [comment]
Additionally, when you have Dungeon Deserter, you can't queue BG (like it works for BG and DF).

**Comment by Alinous0** [comment]
I think it's 15 mins, countdown should start when we enter dungeon.

**Comment by Milton3450** [comment]
Arcadia713 wrote: 'Additionally, when you have Dungeon Deserter, you can't queue BG (like it works for BG and DF).'

I'm pretty sure it shouldn't be like that? I mean, you should be able to queue BG with dungeon deserter and queue for a dungeon with PvP deserter?

**Comment by Arcadia713** [comment]
Yeah, that's why i wrote thats happen :P 
It's need to be fixed (or if it's already, i'm not playing on Atlantiss anymore, so don't know).

**Comment by Aival94** [comment]
The [Dungeon Deserter](http://www.wowhead.com/spell=71041/dungeon-deserter) debuff should be given to anyone who left from a 4 or 5 man group until the 15 dungeon cooldown is active.

http://www.wowhead.com/spell=71041/dungeon-deserter#comments
